### PR TITLE
rbw: Install git credential helper without an option

### PR DESCRIPTION
**Pull request by ivankovnatsky**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/rbw/default.nix [hunk 1]
```diff
@@ -24,7 +24,7 @@
 , pass
 
   # git-credential-helper
-, withGitCredential ? false
+, withGitCredential ? true
```
Comment (marsam) [inline pkgs/tools/security/rbw/default.nix:27]: I don't think we need an option for this, we can install it always.

**Review by marsam — APPROVED** [review]
LGTM. Thank you!

File: pkgs/tools/security/rbw/default.nix [hunk 2]
```diff
@@ -83,7 +80,7 @@ rustPlatform.buildRustPackage rec {
     cp bin/rbw-rofi $out/bin
   '' + lib.optionalString withPass ''
     cp bin/pass-import $out/bin
-  '' + lib.optionalString withGitCredential ''
+  '' + ''
     cp bin/git-credential-rbw $out/bin
```
Comment (SuperSandro2000) [inline pkgs/tools/security/rbw/default.nix:84]: Why not move that to line 76?

File: pkgs/tools/security/rbw/default.nix [hunk 3]
```diff
@@ -61,7 +58,7 @@ rustPlatform.buildRustPackage rec {
     patchShebangs bin/pass-import
     substituteInPlace bin/pass-import \
         --replace pass ${pass}/bin/pass
-  '' + lib.optionalString withGitCredential ''
+  '' + ''
     patchShebangs bin/git-credential-rbw
     substituteInPlace bin/git-credential-rbw \
         --replace rbw $out/bin/rbw
```
Comment (SuperSandro2000) [inline pkgs/tools/security/rbw/default.nix:64]: Lets move that to the beginning before any conditional stuff.

Comment (ivankovnatsky) [reply pkgs/tools/security/rbw/default.nix]: thanks.

File: pkgs/tools/security/rbw/default.nix (hunk 2 above)
Comment (ivankovnatsky) [reply pkgs/tools/security/rbw/default.nix]: thanks.

**Review by luc65r — APPROVED** [review]

**Pull request merged**